src/main/java/ch/njol/skript/expressions/ExprAltitude.java

```diff
@@ -40,11 +40,10 @@
 	static {
 		register(ExprAltitude.class, Double.class, "altitude[s]", "locations");
 	}
-	
-	@SuppressWarnings("null")
+
 	@Override
-	public Double convert(final Location l) {
-		return l.getY();
+	public Double convert(final Location location) {

```

Comment:
l -> location?

Reply:
more proper naming details.

Reply:
Proper naming? Java is statically typed language. If a location is called location, that is not any more informative than calling it just l.